### ISSUE TITLE
Used on the BBC website!

### ISSUE THREAD

**Issue by rtpHarry** [dwayneparton/jquery.scrollmation.js#4]
Did you know your code is being used on the BBC website?

(It's one of the main TV networks if you're not aware / from the UK)

I found this repo by looking at the source code of this page:

https://www.bbc.co.uk/news/resources/idt-sh/stormzys_big_year

**Comment by chavab1** [comment]
@rtpHarry Yeah, I think BBC is using [this](https://shorthand.com/) product which uses scrollmation for their scroll animations. I found this repo by looking at the source code on their sample pages.

**Comment by dwayneparton** [comment]
Oh that's cool. Thanks for the pull requests too!
